### PR TITLE
feat(hx-text-control): add help and error text

### PR DESCRIPTION
**Pull request by BY00565233**
## Description

Add Help Text and Error Text to `hx-text-control`

Jira: SURF-2065

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

**Review by lalithkarikelli — APPROVED** [review]
DEV LGTM

**Review by 100stacks — CHANGES_REQUESTED** [review]
@BY00565233 did you validate this change in all five browsers that we support?

**Comment by BY00565233** [comment]
> @BY00565233 did you validate this change in all five browsers that we support?

File: src/scss/components/text-input/_index.scss [hunk 1]
```diff
@@ -8,12 +8,16 @@
 ///  +----------+---------+----------+
 ///  | [prefix] | control | [suffix] | auto
 ///  +----------+---------+----------+
+///  | help     | help    | help     | auto
+///  +----------+---------+----------+
+///  | error    | error   | error    | auto
+///  +----------+---------+----------+
```
Comment (100stacks) [inline src/scss/components/text-input/_index.scss:14]: Did you manually test this across all five browsers?

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/text-input/_index.scss (hunk 1 above)
Comment (BY00565233) [reply src/scss/components/text-input/_index.scss:14]: I have tested all five browsers(Chrome, Firefox, IE11, legacy Edge and Safari). It's working fine.

**Review by 100stacks — APPROVED** [review]

**Pull request merged**